acceptance/suites/tests/metrics/collect_default_metrics.rb

```diff
@@ -0,0 +1,108 @@
+test_name 'Default metrics are exported to graphite server'
+
+graphite = agents.find { |agent| agent['platform'] =~ /el-6-x86_64/ }
+skip_test 'This test requires an el6 agent' unless graphite
+
+puppetserver_conf_file = options['puppetserver-config']
+metrics_conf_file = "#{options['puppetserver-confdir']}/metrics.conf"
+puppetservice = options['puppetservice']
+environments_dir = "/etc/puppetlabs/code/environments"
+manifests_dir = "#{environments_dir}/production/manifests"
+sitepp = "#{manifests_dir}/site.pp"
+
+step 'Backup puppetserver config files' do
+  on master, "cp -f #{puppetserver_conf_file} #{puppetserver_conf_file}.bak"
+  on master, "cp -f #{metrics_conf_file} #{metrics_conf_file}.bak"
+end
+
+teardown do
+  on master, "mv -f #{puppetserver_conf_file}.bak #{puppetserver_conf_file}"
+  on master, "mv #{metrics_conf_file}.bak #{metrics_conf_file}"
+  if !master.is_pe?
+    on master, "rm -f #{sitepp}"
+  end
+  bounce_service(master, puppetservice)
+end
+
+if !master.is_pe?
```

Comment:
Yeah, good question.  Basically, I had cased this out for PE because it didn't seem to be necessary - because for PE, we should have some function metrics exercised as part of a typical PE agent run anyway - and because I didn't want to have the test dirty up the code dir if run when file-sync is turned on, potentially causing some downstream test failures as a result.  This isn't obvious, though, so I added a comment in 993f5ed to hopefully explain this better.